gnuradio-runtime/lib/math/qa_sincos.cc

```diff
@@ -43,14 +40,12 @@ qa_sincos::t1()
 
     gr::sincos(x, &gr_sin, &gr_cos);
 
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(c_sin, gr_sin, 0.0001);
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(c_cos, gr_cos, 0.0001);
+    BOOST_CHECK_CLOSE(c_sin, gr_sin, 0.0001);
+    BOOST_CHECK_CLOSE(c_cos, gr_cos, 0.0001);
```

Comment:
since gr::sincos is but an alias for `::sincos` if that exists, or for `::sin, ::cos` if it doesn't... testing against an error of 10⁻⁴ seems kind of tame. (`sincos` is a GNU extension of math.h and not C++. It probably has no advantage over separate sin and cos, if you ask me).

But don't necessarily change that here. Noone but gnuradio/math/nco.h's `nco::sincos` and one fpll in gr-atsc uses gr::sincos, anyway, sooo... _cruft_; let's get rid of gr::sincos alltogether in a separate PR.  Why? Because we have VOLK, and `gr::blocks::vco_c` should *really* be using that, and not the unvectorized `gr::math::nco::sincos`.